app/models/mixins/miq_web_server_worker_mixin.rb

```diff
@@ -105,12 +105,12 @@ def self.sync_workers
       result
     end
 
-    def self.thin_pid_file(port)
-      Rails.root.join("tmp/pids/thin.#{port}.pid")
+    def self.pid_file(port)
+      Rails.root.join("tmp/pids/puma.#{port}.pid")
```

Comment:
Nah, let's keep the methods generic... maybe something like this:

If `server thin` is also valid, we can set a constant in the module:

```ruby
RAILS_SERVER = "puma"
...

def self.rails_server_command_line
  @rails_server_command_line ||= begin
    "#{nice_prefix} #{Rails.root.join("bin", "rails")} server #{RAILS_SERVER}".freeze
end
...

def self.pid_file(port)
  Rails.root.join("tmp/pids/rails_server.#{port}.pid")
end
```